### ISSUE TITLE
[EN] [Priest] [General] Psychic scream range [Gamebreaking] 

### ISSUE THREAD

**Issue by Heroclassx** [Atlantiss/BugTracker#10193]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: Should fear up to 5 enemys in 8 yards around the priest. 

**Current behaviour**: Psychic scream range is too short - tooltip ingame says 0 yards but that is probably just display bug? - 

**Expected behaviour**: https://www.wowhead.com/spell=8122/psychic-scream 8 yard range

**Steps to reproduce the problem**:

1. Get an addon to check range 
2. stay at max fear range of the target
3. use fear
4. watch how nothing happens

**Revision**: Current server version (you can see current server revision by typing ".server info" command in the game chat)

**Comment by Aloise3** [comment]
I seem to have an alright range on Psychic Scream. Most addons that measure distance are only approximations (Rangedisplay afaik) and does not depend on the mobs combatreach. Which might be a reason why you’re not hitting an enemy even though the addon displays that you’re 8 yards away.

The 0 yard tooltip is a visual bug.

**Comment by Heroclassx** [comment]
Hmm weird, because I've been playing on around 5 other private servers and also retail priest and never had an issue with missing that many fears :D Also from other priest players I hear complains about it.

**Comment by Aloise3** [comment]
I don’t play a lot of priest, especially not in PvP. But in dungeons and random mobs the range works alright.
I just listed a couple a reasons why it could differ from retail, but it’s close to 8 yards. Could be some server lag combined with movement, I don’t know. If stuff stands still it’s definetly about 8 yards range

**Comment by Axelitoo** [comment]
Played priest a bit in PvP, Scream range does seem very, very fucked.

**Comment by Illusie** [comment]
It's indeed seems fucked I remember myself need to stand "on" target himself to fear him.
I think it would be hard to find retail video of Priest using fear in fair distance in pvp.

**Comment by Aloise3** [comment]
So it is a pvp issue? @heroclassx does it only happen when there are plenty of enemies or also 1v1? Some specifics would probably help the devs.

**Comment by Heroclassx** [comment]
Yeah I know this report is kinda "weak" but I cant think of anything to trigger the problem. Sometimes it just feels so off with fear range. It happens in 1v1 same as plenty. I can link some youtube videos where you can see the Fear range of fears that hit while they dont hit on atlantiss, I doubt that would be helpful tho. :D

**Comment by Genomnomnomnom** [comment]
Friend of mine got the same problem randomly. It might be the same random range issue as getting Hunter's traps by friends.
